packages/@jsii/spec/src/assembly-utils.ts

```diff
@@ -40,6 +40,30 @@ export function findAssemblyFile(directory: string) {
   return dotJsiiFile;
 }
 
+/**
+ * Replaces the file where the original assembly file *should* be found with a new assembly file.
+ * Detects whether or not there is a compressed assembly, and if there is, compresses the new assembly also.
+ * Replaces the fingerprint with '**********' rather than recalculating it, since we have modified the assembly.
+ */
+export function replaceAssembly(assembly: Assembly, directory: string) {
+  writeAssembly(directory, _fingerprint(assembly), {
+    compress: compressedAssemblyExists(directory),
+  });
+}
+
+/**
+ * Replaces the old fingerprint with '***********'.
+ *
+ * @rmuller says fingerprinting is useless, as we do not actually check
```

Comment:
That means nothing here... if you're gonna quote me, at least use my GitHub alias... Also I don't think I've ever said "useless", probably "currently not used", and so it was future-proofing...

Reply:
Whoops... somehow it came out of the previous closed PR and I have no idea how I landed on that comment anyway. Will simply remove your alias entirely.